doctrine.rst

```diff
@@ -62,14 +63,50 @@ Creating an Entity Class
 
 Suppose you're building an application where products need to be displayed.
 Without even thinking about Doctrine or databases, you already know that
-you need a ``Product`` object to represent those products. Use the ``make:entity``
-command to create this class for you:
+you need a ``Product`` object to represent those products.
+
+.. _doctrine-adding-mapping:
+
+You can use the ``make:entity`` command to create this class and any fields you
+need. The command will ask you some questions - answer them like we've done below:
```

Comment:
We never use "we" in the docs, so what about: `like we've done below` -> `like done below` ?